Automattic/wp-calypso
# Activity log: remove filterbar icon

This PR removes the icon from the filterbar in Activity.

### Before

![image](https://user-images.githubusercontent.com/390760/45886703-b34f9b80-bdb1-11e8-99ac-520eab2848a2.png)

![image](https://user-images.githubusercontent.com/390760/45886741-ca8e8900-bdb1-11e8-97d7-c84d763f838c.png)

### After

![image](https://user-images.githubusercontent.com/390760/45886715-bb0f4000-bdb1-11e8-8347-fe0405a58474.png)

![image](https://user-images.githubusercontent.com/390760/45886761-d5491e00-bdb1-11e8-8161-f5bf9c2cd5d6.png)

### How to test

- Run locally or fire up https://calypso.live/?branch=remove/activity-log-filterbar-icon
- Go to Activity.
- Ensure the icon is gone and that the label works fine.
